### ISSUE TITLE
Menue Item misplaced

### ISSUE THREAD

**Issue by hitam4450** [owncloud/contacts#297]
The 'Action' menu item  **import contacts **is located in the settings menu, which is not intuitive at all and logically not correct, to place an '**action**' item in the '**settings**' menu!

The same holds true in the calendar app !! (https://github.com/owncloud/calendar/issues/348)

regards, hitam4450

### Steps to reproduce

1.
2.
3.
### Expected behaviour

Tell us what should happen
### Actual behaviour

Tell us what happens instead
### Server configuration

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**ownCloud version:** (see ownCloud admin page)

**Contacts version:** (see ownCloud apps page)

**Updated from an older ownCloud or fresh install:**

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```

**The content of config/config.php:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your ownCloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
### Client configuration

**Browser:**

**Operating system:**

**CardDAV-clients:**
#### Web server error log

```
Insert your webserver log here
```
#### ownCloud log (data/owncloud.log)

```
Insert your ownCloud log here
```
#### Browser log

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by skjnldsv** [comment]
Hello, thanks for your feedback.
Where do you suggest we should put the import button?

**Comment by hitam4450** [comment]
I would suggest a prominent location on the left pane e.g. in the top part, so it is visible to everyone, no search is required, no support questions asked! 
KISS= Keep it Simple Stupid!

regards, hitam4450

**Comment by gvde** [comment]
I wouldn't place the import function into such a prominent place. It's not something to be used on a daily basis thus putting it into the menu is the perfect place for me.

Instead, I wouldn't call that menu 'Settings' as sharing, url view, deleting address books is not really a "setting" to begin with. I would rather call that menu something like "Advanced" or "Tools" which is more like what it contains...

**Comment by hitam4450** [comment]
Hi, everything menu item different  then Settings is Ok with me........

However, the solution should be consistent across all OC apps: calendar, music, gallery, contacts, etc.

**Comment by skjnldsv** [comment]
Maybe adding an action bar just under the Settings label?
But we would need more option to put in that bar, just having he "import"  link isn't good enough.

**Comment by Henni** [comment]
I also thought about placing it like the "Deleted files" in the files app:
![image](https://cloud.githubusercontent.com/assets/1267827/14059974/d76303d8-f354-11e5-8b2e-922f8df93893.png)

But in the end I believe the right place is the settings menu, because this is also where the export function will be.

Maybe it would make sense to rename "Settings" to "Addressbooks" and choose a different icon.

**Comment by Henni** [comment]
@owncloud/designers where should we place the Import feature in your opinion?

**Comment by DeepDiver1975** [comment]
Having Import down in the settings isfine for me. Its not a feature you use every day.

**Comment by skjnldsv** [comment]
Same here!

**Comment by thorsten19** [comment]
Having the import function in the settings section is the right place. This is nothing a user will do frequently.
I suggest to only display an icon that uses the arrow in up direction (similar to what you currently use next to the Import text) and to place it in the same row wher you currently have the 'Show Url' and 'Share Addressbook' icons/links. Then you do do not have any more issues for this like reported in issue #290.

**Comment by jancborchardt** [comment]
Folks – look at the old Contacts app. For the initial empty experience, the import button is big and fat inside the empty state. That is where it belongs and we need to do the same again.

Same for Calendar actually cc @georgehrke @raghunayyar

**Comment by jancborchardt** [comment]
(And yes, it should also be in settings for any subsequent need for importing.)

**Comment by georgehrke** [comment]
@jancborchardt Concerning the calendar app - What about:
If there there is no writable calendar, show an `Import calendar` button below the `new calendar` button?

**Comment by jancborchardt** [comment]
@georgehrke yeah, sounds good. (Forgot that there is not really an empty state since we have the Calendar grid.)

**Comment by skjnldsv** [comment]
So, what are we deciding?
Should we merge #279 and wait for a language to have a bigger word and moving the button?
Or should we wait for more action to make an action menu inside the settings? (import +download? +fix duplicate?)

**Comment by hitam4450** [comment]
I vote for to make a separate "Action menu", where all "action" items are grouped together!

regards, hitam4450

**Comment by jancborchardt** [comment]
@skjnldsv I’d say let’s just put the import button in its own row. The import icon should also be to the left of the text, we never put it to the right.

**Comment by DeepDiver1975** [comment]
fixed with #402
